exhibits-solr-conf.gemspec

```diff
@@ -0,0 +1,29 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'exhibits-solr-conf/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'exhibits-solr-conf'
```

Comment:
I hate underscores for this stuff (extra keystrokes) and I certainly don't want Exhibits::Solr::Conf - bleah.   This is a really really dumb stupid gem.  Do we really care?